doc_template/biophysical_models.rst

```diff
@@ -101,7 +101,10 @@ This can take some time.
 
     cd neuronal_model
     nrnivmodl ./modfiles   # compile the model (only needs to be done once)
-    python -m allensdk.model.biophysical.runner manifest.json
+    python -m allensdk.model.biophysical.runner --maniest_file manifest.json # perisomatic models
```

Comment:
"maniest_file" -> "manifest_file"

also, is the old positional argument syntax no longer supported?

Reply:
Thanks for the catch. Yes, the old positional argument is not supported with the schema. I used argschema since this is being used in the other modules. I can just use argparse if this is gonna be a problem.

Reply:
well, if we break the old interface we need to reflect that in a major version increment.

Reply:
seems not worth it here though

Reply:
cool..changing it to argparse